### DISCUSSION TITLE
My suggestions (feedback)

### DISCUSSION THREAD

**Discussion by drogga** [anilbeesetti/nextplayer#659] [Ideas]
Can there please be greater/higher playback speeds than 4.0 & up-to 7.0 ?
Most video players fail in this and the A/V gets out of sync and either the video, but mostly the audio finishes first and you watch only the video until it plays out, unlike in a browser for example with a playback speed extension/add-on, where I can go x13 if I want and if the device can handle it, the A/V are in sync and it doesn't fast play frame by frame, but as is, just a lot faster.

_And before someone asks why I want or need such high speeds > I have my reasons. I also play some YT videos at x6.0 with SkyTube_

**Comment by drogga** [comment]
You might also consider adding a settings button,  so I can jump to the preferences to adjust something without having to close the currently playing media, especially if it's a online streaming one.

**Reply by drogga** [reply]
If it's not too much work for nothing, then the settings can also be added as a quick jump shortcut action by long press on the app icon, those that are listed below, if you know what I mean.., if you don't, then long-tap/press/hold on the YT app icon/shortcut, Chrome or others, the available options shortcuts should appear below and can be dragged to make shortcuts of them.

**Reply by drogga** [reply]
Yesterday I played a portrait video DLed from somewhere, which I then deleted and couldn't reproduce again today with one that I recorded, the issue: SGS8+ (A.9 Pie) connected to a TV (screen mirror, not Dex mode), the video looked all good with other players I have, but with NextPlayer it did push the contrast for some reason, which was easy to see in landscape, because the device's aspect ration isn't 16:9, but 18.5:9 (taller) and the TV ofc. is 16:9, so there were top & bottom true black lines, but the side ones were with punched contrast, unless I hold (long click for 2x speed) with the BT mouse, then it looked as it should, the N.P. ver. was 0.9.0 and tested the same today with that one and then updated the latest 0.10.1, even tho nothing is mentioned in the changes since that one regarding or fixing this, it's either the video (don't have it anymore or can find it - no way) or the mouse cursor that is bugging this out, today I tested without a mouse.

**Comment by anilbeesetti** [comment]
@drogga, numerous videos are currently not functioning properly, even at 4x speed, let alone 7x speed. If I introduce 7x speed now, many users will complain that it's not functioning. Therefore, I must address the issues with 4x speed first before considering higher playback speeds.
